qualibration_graphs/nv_center/quam_config/generate_quam.py

```diff
@@ -0,0 +1,82 @@
+"""
+
+import matplotlib.pyplot as plt
+from qualang_tools.wirer.wirer.channel_specs import *
+from qualang_tools.wirer import Instruments, allocate_wiring, visualize
+from utils.connectivity_nv_center_interface import Connectivity
+from quam_builder.builder.qop_connectivity import build_quam_wiring
+from quam_builder.builder.nv_center.build_quam import build_quam
+from my_quam import Quam
+
+########################################################################################################################
+# %%                                              Define static parameters
+########################################################################################################################
+host_ip = "127.0.0.1"  # QOP IP address
+port = None  # QOP Port
+cluster_name = "Cluster"  # Name of the cluster
+
+########################################################################################################################
+# %%                                      Define the available instrument setup
+########################################################################################################################
+instruments = Instruments()
+instruments.add_mw_fem(controller=1, slots=[1, 2])
+instruments.add_lf_fem(controller=1, slots=[3])
+
+########################################################################################################################
+# %%                                 Define which qubit ids are present in the system
+########################################################################################################################
+qubits = [1]
+# qubit_pairs = [(qubits[i], qubits[i + 1]) for i in range(len(qubits) - 1)]
+
+########################################################################################################################
+# %%                                 Define any custom/hardcoded channel addresses
+########################################################################################################################
+# qubit drive channel
+q1_drive_ch = mw_fem_spec(con=1, slot=1, out_port=2)
+# qubit laser channel
+# Todo: add digital-only channels
+# q1_laser_ch = ChannelSpecLfFemDigital(con=1, slot=3, out_port=3)
+# qubit readout (SPCM) input channel
+q1_spcm1_in_ch = lf_fem_spec(con=1, out_slot=3, out_port=2, in_slot=3, in_port=1)
+# q1_spcm2_in_ch = lf_fem_spec(con=1, out_slot=3, out_port=4, in_slot=3, in_port=2)
+
+########################################################################################################################
+# %%                Allocate the wiring to the connectivity object based on the available instruments
+########################################################################################################################
+connectivity = Connectivity()
+# The readout lines
+# connectivity.add_laser(qubits=qubits, constraints=q1to2_laser_ch)
```

Comment:
I added the laser as a digital output

Reply:
Yes, thanks. It should be properly handled now.